File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/SykefraværsstatistikkNæringMedVarighet.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.importering;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class SykefraværsstatistikkNæringMedVarighet implements Sykefraværsstatistikk {
  private int årstall;
  private int kvartal;
  private String næringkode;
  private String varighet;
  private int antallPersoner;
  private BigDecimal tapteDagsverk;
  private BigDecimal muligeDagsverk;
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/fellesdomene/SykefraværsstatistikkNæringMedVarighet.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene

import java.math.BigDecimal

data class SykefraværsstatistikkNæringMedVarighet(
    override val årstall: Int = 0,
    override val kvartal: Int = 0,
    val næringkode: String? = null,
    val varighet: Char? = null,
    override val antallPersoner: Int = 0,
    override val tapteDagsverk: BigDecimal,
    override val muligeDagsverk: BigDecimal,
) : Sykefraværsstatistikk

